src/js/components/header/index.js

```diff
@@ -0,0 +1,22 @@
+var yo = require('yo-yo')
+var header = require('dat-header')
+
+module.exports = Header
+
+function Header (el, props) {
+  if (!(this instanceof Header)) return new Header(el, props)
+  this.$el = document.getElementById(el)
+  this.props = props
+  this._component = this._render()
+  if (this.$el) this.$el.appendChild(this._component)
+}
+
+Header.prototype.update = function (state) {
+  yo.update(this._component, this._render())
+}
+
+Header.prototype._render = function () {
+  var self = this
+  var props = self.props
+  return yo`${header(props)}`
```

Comment:
am pretty sure it should be `${header(this.props)}` 

(and per comment below, since `this.props` should not be made publicly available, it should be marked as private by calling it `this._props`).

so final answer: `${header(this._props)}`